currency-rates/internal/services/mail.go

```diff
@@ -1,42 +1,87 @@
 package services
 
 import (
+	"crypto/tls"
 	"net/smtp"
 
 	"go.uber.org/zap"
 )
 
+const serverName = "smtp.gmail.com"
```

Comment:
It would be nice to replace hardcoded value with one from environment config.

Reply:
Values that are not expected to change and are not sensitive data can be stored as constants